src/components/shared/StepWidget.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { TiTick } from 'react-icons/ti';
+import '../../assets/styles/components/stepper.scss';
+
+type stepperProps = { steps: string[] };
+
+function Stepper({ steps }: stepperProps) {
+  const [currentStep, setCurrentStep] = useState(1);
+  const [complete, setComplet] = useState(false);
+
+  function next() {
+    if (currentStep === steps.length) setComplet(true);
+    else {
+      setCurrentStep((prev: number) => prev + 1);
+    }
+  }
+
+  function previous() {
+    if (currentStep > 1) {
+      setComplet(false);
+      setCurrentStep((prev: number) => prev - 1);
+    }
+  }
+
+  return (
+    <div className="absolute top-[10vh] left-[0vw] flex sm:flex-row flex-col  justify-between">
```

Comment:
Is this wise though? Using viewports to position a component that could be used anywhere in the app?

I think because we don't know where this component will be used, you can just position it without relying on the viewports. i.e `vh` and `vw`. I think simple integers could do the trick for it. But keep in mind that that `absolute` position will only work if the parent of this widget has a `relative` position. Just saying.

But it looks great. 

P.S. Remember to take a look at the reviews on chromatic as well.